src/msh2gprs/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(msh2gprs LANGUAGES CXX)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+find_package(Qt5 COMPONENTS Widgets REQUIRED)
+
+add_executable(msh2gprs
+  main.cpp
+  mainwindow.cpp
+  mainwindow.h
+  mainwindow.ui
+)
+
+target_link_libraries(msh2gprs PRIVATE Qt5::Widgets)
```

Comment:
Why do you need QT???

Reply:
On Linux, I used QT to debug MSH2GPRS. It is better to remove QT parts.  I didn't recognize qt-related part is added to CMakeList. I will modify this and pull request again.